contrib/cirrus/lib.sh

```diff
@@ -181,10 +181,9 @@ in_podman() {
     done
     showrun podman run -i --name="$IN_PODMAN_NAME" \
                    --net="container:registry" \
-                   --security-opt=label=disable \
-                   --security-opt=seccomp=unconfined \
-                   --cap-add=all \
                    --env-file=$envfile \
+                   --privileged \
+                   --cgroupns=host \
```

Comment:
I think PR works without `--cgroupns=host` do we need this for any special reason.

Reply:
I think it is good to add it.  We are bind mounting `/sys/fs/cgroup` from the host, so the expectation is that the same cgroups are visible within the container

Reply:
ah I see, got it.